src/utils/isReactComponentClass.js

```diff
@@ -44,6 +44,27 @@ export default function isReactComponentClass(
     return true;
   }
 
+  // check for @extends React.Component in docblock
+  if (path.parentPath && path.parentPath.value) {
+    var classDeclaration;
+    if (Array.isArray(path.parentPath.value)) {
+      var matches = path.parentPath.value.filter(function(declaration) { return declaration.type === 'ClassDeclaration' });
```

Comment:
Could use `.find` here fwiw, which would let you make this a one liner

``` js 
var classDeclaration = Array.isArray(path.parentPath.value)
  ?  path.parentPath.value.filter(function(declaration) { return declaration.type === 'ClassDeclaration' });
  : path.parentPath.value

Reply:
Nice! So much cleaner.